docs/Upgrade_Suggestions_Data_Analysis.md

```diff
@@ -0,0 +1,87 @@
+# Upgrade Suggestion (Data Analysis)
+
+Upgrading old Chef infrastructure could be very challenging, in former times,
+users would upload cookbooks and register Chef nodes (bootstrap) to a Chef
+Infra Server, such nodes would have a defined run-list, that is, the list of
+policies (cookbooks) applied to the node. The run-list could also be modified
+by roles and environments, giving the user a lot of flexibility, but with a
```

Comment:
I was referring to that exactly. You specify a run-list that 100 nodes could share, but 10 are in dev, 30 are in QA and 60 are in production and so, the version of the cookbooks may differ. 

I will update this so that it is better explained. 👍